### PR TITLE
Add multiple product images in the feedspec

### PR DESCRIPTION
**Pull request by anastasop**

### CODE REVIEW

**Comment by fotos** [comment]
+1 LGTM

**Comment by fotos** [comment]
@anastasop the turkish translation is missing on purpose?

**Comment by anastasop** [comment]
@fotos pending

File: source/localizable/feedspec/index.el.html.md.erb [hunk 1]
```diff
@@ -230,6 +231,29 @@ thumbnail.
...
+
+Το πεδίο είναι προαιρετικό αλλά συνίσταται η χρήση του. Μπορεί να εμφανίζεται
+πολλές φορές σε ένα feed, μιά φορά για κάθε διαφορετική φωτογραφία.
+Ισχύουν οι ίδιοι περιορισμοί με το [Image Link](#image-link)
+
+##### Παράδειγμα
+
+~~~ xml
+<image>http://www.mydomain.com/photos/1.jpg</image>
+<imageurl>http://www.mydomain.com/photos/b&amp;amp;b.jpg</imageurl>
```
Comment (Zorbash) [inline source/localizable/feedspec/index.el.html.md.erb]: Are you sure you want this URL to be displayed as `http://www.mydomain.com/photos/b&amp;amp;b.jpg`?

**Comment by Zorbash** [comment]
+1 LGTM

**Pull request merged**